src/App/Documentation/components/Panel/index.js

```diff
@@ -59,17 +61,35 @@ const Overview = () => (
     <>
         <h2 id="overview">Overview</h2>
         <p>
-            A panel consists of three parts: A header, a body and a footer. You can mostly put whatever you want in any of these,
-            but try not to put too much in the header and footer as these are meant to contain short and concise information about the body content.
+            A panel consists of three parts: A header, a body and a footer.
+            You can mostly put whatever you want in any of these, but try not to put too much in the header and footer as these are meant to contain short and concise information about the body content.
         </p>
         <ComponentPreview language="html" showCasePanel codeFigure>
             <PanelComponent
                 title="Panel title"
                 subTitle="Panel subtitle"
                 footerText="Footer content goes here."
                 footerBtnText="Footer button"
-                bodyContent={bodyContent}
-            />
+            >
+                {bodyContent}
+            </PanelComponent>
+
+        </ComponentPreview>
+        <p>
+            You can choose whether to use <Property value=".panel-header" /> and <Property value=".panel-footer" /> or not.
+            When using the <PrismCode className="language-html">{"<footer/>"}</PrismCode> or <PrismCode className="language-html">{"<header/>"}</PrismCode> they need to be direct children of <Property value=".panel" />.
```

Comment:
What do we recommend? Most of our user do not have the knowledge to take these kind of decisions and is not experiencing the issue I had. Is it a possible solution move it to the bottom of the page and present it as an alternate markup when the DesignGuide user can't have the footer and header as a direct child?

Reply:
Well, if we want to recommend the simplest markup then we need to explain this to them. I also feel like we should inform them about this possible issue as soon as possible so they don't spend time trying to figure out why their panel is not working like ours. Can we not expect our users to understand descendant selectors?

Reply:
But why should we in the Overview section explain something that does not apply to most users, and for them possibly create confusion. Is it many people experiencing this issue? Is this not an edge case?

> Can we not expect our users to understand descendant selectors?
A portion of them probably don't.

Another possibility is just to remove the doc for the footer and header without class, and only keep it for backwards compatibility.

Reply:
Ok, does this mean using footers and headers without classes is deprecated? Or do we just do a "silent" change but keep the classless versions for backward compat.

Reply:
I guess so.